src/components/accession2/properties/CollectedReceivedDate2.tsx

```diff
@@ -42,22 +51,22 @@ export default function CollectedReceivedDate2({ onChange, record, type, validat
 
   const validateDate = useCallback(
     (id: string, value?: any) => {
-      const date = new Date(value);
+      const date = DateTime.fromISO(value, { zone: timeZone });
```

Comment:
there is a fromJS in luxon, if you want to keep web-components returning the JS object like it did previously.. it has more info than the string

Reply:
fwiw, we can keep original code as-is, here.
the only thing we need to do is pass the timeZone on line 62
there's no need to convert to luxon in this use-case

Reply:
i take that back, this change is a good thing if we are going to move to luxon throughout anyways. 
we need to update Datepicker to return luxon date as well, and move away from momentjs. that's a breaking/bigger change.. for another time.

Reply:
Changed it to receive a string when validating